fe1-web/src/features/digital-cash/__tests__/utils.ts

```diff
@@ -0,0 +1,96 @@
+import { mockKeyPair, mockLaoIdHash } from '__tests__/utils';
+import { Hash, PopToken, RollCallToken } from 'core/objects';
+import { COINBASE_HASH, SCRIPT_TYPE } from 'resources/const';
+
+import {
+  DIGITAL_CASH_FEATURE_IDENTIFIER,
+  DigitalCashFeature,
+  DigitalCashReactContext,
+} from '../interface';
+import { TransactionJSON, TransactionState } from '../objects/transaction';
+
+export const mockRollCall = {
+  id: new Hash('rcid'),
+  name: 'rc',
+  attendees: [mockKeyPair.publicKey],
+  containsToken: () => true,
+} as DigitalCashFeature.RollCall;
```

Comment:
For tests inter-feature dependencies are allowed so it would also be possible to just re-use the mock roll call from the roll call feature :)